Content.Server/Chemistry/ReagentEffects/ChemHealEyeDamage.cs

```diff
@@ -1,27 +1,28 @@
 using Content.Shared.Chemistry.Reagent;
 using Content.Shared.Eye.Blinding;
+using Content.Shared.Eye.Blinding.Systems;
 using JetBrains.Annotations;
 
 namespace Content.Server.Chemistry.ReagentEffects
 {
     /// <summary>
-    /// Heal eye damage (or deal)
+    /// Heal or apply eye damage
     /// </summary>
     [UsedImplicitly]
     public sealed class ChemHealEyeDamage : ReagentEffect
     {
         /// <summary>
-        /// How much eye damage to remove.
+        /// How much eye damage to add.
         /// </summary>
         [DataField("amount")]
         public int Amount = -1;
 
         public override void Effect(ReagentEffectArgs args)
         {
-            if (args.Scale != 1f)
+            if (args.Scale != 1f) // huh?
                 return;
```

Comment:
I'm not sure what this is meant to do? Is it supposed to be a quantized reaction?